docs/src/_data/rules_category.js

```diff
@@ -0,0 +1,32 @@
+let pathPrefix = "/docs/head/";
+
+if (process.env.CONTEXT === "deploy-preview") {
+    pathPrefix = "/";
+} else if (process.env.BRANCH === "latest") {
+    pathPrefix = "/docs/latest/";
+} else if (process.env.BRANCH === "next") {
+    pathPrefix = "/docs/next/";
+}
```

Comment:
To avoid duplicating this logic here, we can add the path prefix to global data and use it here.

Here's an example:

https://github.com/eslint/eslint/blob/1000187e00949332babcee4d37d46c96a6a554a8/docs/.eleventy.js#L53

https://github.com/eslint/eslint/blob/1000187e00949332babcee4d37d46c96a6a554a8/docs/src/_data/site.js#L18-L20